src/System.Threading/ref/System.Threading.cs

```diff
@@ -324,6 +330,24 @@ public partial class SynchronizationLockException : System.Exception
         public SynchronizationLockException() { }
         public SynchronizationLockException(string message) { }
         public SynchronizationLockException(string message, System.Exception innerException) { }
+        protected SynchronizationLockException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+    }
+    public sealed partial class ThreadAbortException : System.SystemException
```

Comment:
It may be more appropriate for ThreadAbortException, ThreadExceptionEventArgs and ThreadExceptionEventHandler to live in System.Threading.Thread